packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -2,36 +2,92 @@ import React, { ReactElement } from 'react';
 import classNames from 'classnames';
 import { ChecklistCardComponent } from '../cards/Card';
 import { ChecklistStep } from './ChecklistStep';
-import { ChecklistCardProps } from '../../lib/checklist';
+import {
+  ChecklistCardProps,
+  ChecklistCardVariant,
+  ChecklistVariantClassNameMap,
+} from '../../lib/checklist';
 import { useChecklist } from '../../hooks/useChecklist';
-import { RankConfetti } from '../../svg/RankConfetti';
+import { ChecklistConfetti } from './ChecklistConfetti';
+
+const containerClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'rounded-14',
+  [ChecklistCardVariant.Small]: 'rounded-10',
+};
+
+const headerClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'p-4 rounded-t-12',
+  [ChecklistCardVariant.Small]: 'p-2 rounded-t-8',
+};
+
+const headerClassNameClosedMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'p-4 rounded-12',
+  [ChecklistCardVariant.Small]: 'p-2 rounded-8',
+};
+
+const titleSizeToClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'typo-body',
+  [ChecklistCardVariant.Small]: 'typo-footnote',
+};
+
+const stepsContainerClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'p-4',
+  [ChecklistCardVariant.Small]: 'p-2',
+};
+
+const progressClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'mt-6 gap-2',
+  [ChecklistCardVariant.Small]: 'absolute left-0 right-0 top-0 z-3',
+};
+
+const progressItemClassNameMap: ChecklistVariantClassNameMap = {
+  [ChecklistCardVariant.Default]: 'h-3 rounded-6 bg-white',
+  [ChecklistCardVariant.Small]: 'h-0.5 bg-white',
+};
 
 const ChecklistCard = ({
   className,
   title,
-  description,
+  content,
```

Comment:
`content` is now `ReactNode` and replaces description. It allows to customize extension content above steps. Updated tests and squad implementation.